server/capture.c

```diff
@@ -448,6 +450,7 @@ static void gather_data_capture(
 {
     /* Work through the fields. */
     struct data_capture *capture = gather->capture;
+    capture->sample_count_anonymous = false;
```

Comment:
I'm not happy about putting this here.  The problem is that you're assigning to `sample_count_anonymous` in two different places, and this line here isn't a full initialisation of `*capture`.  You could perhaps do
```
*capture = (struct data_capture) {};
```
here instead, but I'd want to know that this was the right time and place for this.

Anyway, hopefully we'll talk about this and related issues later today.